src/screens/ProfileScreen/index.js

```diff
@@ -141,9 +137,11 @@ const Header = (props) => {
       <View>
         <View style={styles.tabs} ref={postRef}>
           <CustomPressable
-            style={styles.tabItem(profileTabIndex === TAB_INDEX_SIGNED)}
+            style={styles.tabItem(profileTabIndex === TAB_INDEX_SIGNED, true)}
             onPress={setTabIndexToSigned}>
-            <Text style={styles.postText(profileTabIndex === TAB_INDEX_SIGNED)}>Signed Posts</Text>
+            <Text style={styles.postText(profileTabIndex === TAB_INDEX_SIGNED, true)}>
+              Signed Posts
+            </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `isSigned` parameter to the `styles.tabItem` and `styles.postText` functions is a good way to handle different styles based on whether the tab is signed or not. However, it's important to ensure that all calls to these functions are updated to include this new parameter.

```diff
- style={styles.tabItem(profileTabIndex === TAB_INDEX_SIGNED)}
+ style={styles.tabItem(profileTabIndex === TAB_INDEX_SIGNED, true)}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->